### DISCUSSION TITLE
Figma design system

### DISCUSSION THREAD

**Discussion by cutbypham** [rose-pine/rose-pine-theme#99] [Ideas]
![image](https://user-images.githubusercontent.com/40050527/219338463-719153da-f979-4220-a9e1-ac4966df408b.png)

Link: https://www.figma.com/file/a3GbTTKbuqkTeHiqJi3iYY/Ros%C3%A9-Pine-Design-System?node-id=0%3A1&t=R312vTfIP86E2nok-1

**Comment by cutbypham** [comment]
- [ ] Palette

**Comment by mvllow** [comment]
We do have an internal Figma doc that includes our palette, logos, and much more. I'd love to make this public eventually and potentially add others in to help with it :)
